app/src/main/java/com/plan2meet/ui/MainViewModel.kt

```diff
@@ -1,4 +1,16 @@
 package com.plan2meet.ui
 
-class MainViewModel {
+import androidx.lifecycle.ViewModel
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.FirebaseFirestoreSettings
+import com.google.firebase.storage.FirebaseStorage
+
+class MainViewModel : ViewModel() {
+
+    //firebase
+    private var firestore : FirebaseFirestore = FirebaseFirestore.getInstance()
+    private var storageReference = FirebaseStorage.getInstance().reference
+    init {
+        firestore.firestoreSettings = FirebaseFirestoreSettings.Builder().build()
+    }
```

Comment:
This looks like an incomplete feature.
Are you using Firebase anywhere?  I don't see it used.
Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.